opentreemap/treemap/lib/user.py

```diff
@@ -98,6 +109,11 @@ def get_audits(logged_in_user, instance, query_vars, user, models,
     if exclude_pending:
         audits = audits.exclude(requires_auth=True, ref__isnull=True)
 
+    # Uncomment to dump the large, complex query for debugging
```

Comment:
We don't have a precedent for committing this type of code comment. Over the lifetime of this project we'll profile many queries, and often times once they've been profiled once, they are done. For these two reasons, I concerned that leaving this comment is not a sustainable idea. It seems likely that these comments will contribute noise and will not be well-placed in the long-term.

Keeping this technique on the brain is important though. Consider writing this up in documentation instead, or factoring it out as a helper that takes a queryset, confirms `settings.DEBUG == True`, and does this.